### PR TITLE
Adding web3j deploy <profile-name> <package-name> command

### PR DESCRIPTION
**Pull request by mahima-manik**
### What does this PR do?
Adding web3j deploy <profile-name> <package-name> command

### Where should the reviewer start?
From Web3jCommand.java file

### Why is it needed?
Deploy command

### CODE REVIEW

File: src/main/resources/project/build.gradle.template [hunk 1]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradle.template:7]: this needs to change to `4.8.8`

File: src/main/resources/project/build.gradle.template [hunk 2]
```diff
@@ -34,6 +35,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradle.template:38]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/build.gradleImport.template [hunk 3]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleImport.template:7]: `4.8.8`

File: src/main/resources/project/build.gradleImport.template [hunk 4]
```diff
@@ -33,6 +34,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleImport.template:37]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/build.gradleImportJava11.template [hunk 5]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleImportJava11.template:7]: `4.8.8`

File: src/main/resources/project/build.gradleImportJava11.template [hunk 6]
```diff
@@ -32,6 +33,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleImportJava11.template:36]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/build.gradleJava11.template [hunk 7]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleJava11.template:7]: `4.8.8`

File: src/main/resources/project/build.gradleJava11.template [hunk 8]
```diff
@@ -33,6 +34,7 @@ dependencies {
     implementation("org.web3j:web3j-unit:$web3jVersion")
     implementation("org.web3j:web3j-evm:$web3jVersion")
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/build.gradleJava11.template:37]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/erc20/build.gradleErc20.template [hunk 9]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/erc20/build.gradleErc20.template:7]: `4.8.8`

File: src/main/resources/project/erc20/build.gradleErc20.template [hunk 10]
```diff
@@ -33,6 +34,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/erc20/build.gradleErc20.template:37]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/erc20/build.gradleErc20Java11.template [hunk 11]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/erc20/build.gradleErc20Java11.template:7]: `4.8.8`

File: src/main/resources/project/erc20/build.gradleErc20Java11.template [hunk 12]
```diff
@@ -33,6 +34,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/erc20/build.gradleErc20Java11.template:37]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/erc777/build.gradleErc777.template [hunk 13]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/erc777/build.gradleErc777.template:7]: `4.8.8`

File: src/main/resources/project/erc777/build.gradleErc777.template [hunk 14]
```diff
@@ -33,6 +34,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/erc777/build.gradleErc777.template:37]: This is not required anymore, the plugin injects the dependency.

File: src/main/resources/project/erc777/build.gradleErc777Java11.template [hunk 15]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
     id 'org.web3j' version '4.8.4'
+    id 'org.web3j.deploy' version '0.0.1'
```
Comment (AlexandrouR) [inline src/main/resources/project/erc777/build.gradleErc777Java11.template:7]: 4.8.8

File: src/main/resources/project/erc777/build.gradleErc777Java11.template [hunk 16]
```diff
@@ -29,6 +30,7 @@ dependencies {
     implementation "org.web3j:web3j-unit:$web3jVersion"
     implementation "org.web3j:web3j-evm:$web3jVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.web3j:web3j-deployer:0.0.1"
```
Comment (AlexandrouR) [inline src/main/resources/project/erc777/build.gradleErc777Java11.template:33]: This is not required anymore, the plugin injects the dependency.

**Review by AlexandrouR — COMMENTED** [review]
I would also like to see some tests if there is a way to mock this.

**Comment by mohamedelshami** [comment]
Closing this for now, the deployer branch requires to rebase with the most recent code base.

**Pull request closed without merging**